### DISCUSSION TITLE
Which ice scheme for "Small-domain LES with ice" ?

### DISCUSSION THREAD

**Discussion by jchylik** [ARM-Development/comble-mip#47] [Q&A] [answered]
The outputs of |  SCM/Small-domain LES with ice | {prefix}_FixN | are expected on 1 April.
Which ice scheme are we expected to use here? A scheme native to each model? Or the "diagnostic ice" ?

A related issue is that if we use a 2-moment prognostic bulk scheme for ice, then some of the processes (for example freezing and riming) would reduce the mass of cloud droplets while keeping their number.

**Comment by fridlind** [comment]
This simulation is intended to test SCM results when droplet number concentration is fixed and ice formation is diagnostic. In ModelE3, we use the following approaches. First, when droplet number concentration is used, then we assume that it is the specified fixed concentration. Second, when the conditions for ice formation exist (liquid and temperature thresholds exceeded) and the ice number concentration is also below the specified fixed concentration, then we follow an immersion freezing process to produce sufficient ice to reach the specified fixed concentration. We note that the immersion freezing process would reduce the droplet number concentration if it were permitted to deviate from the fixed value. Other processes such as vapor exchange also change the mass of cloud droplets. Thank you of the question, and please let us know if anything remains unclear.

**Reply by jchylik** [reply]
> droplet number concentration is fixed and ice formation is diagnostic. 

Thank you for the clarification.

> We note that the immersion freezing process would reduce the droplet number concentration if it were permitted to deviate from the fixed value. 

Oh, that was the piece that I was still missing.
Thank you for the guidelines!

> and please let us know if anything remains unclear.

There is still one bit unclear:
If we have a gridpoint where all three are present:
droplets (fixed), raindrops , and lots of sedimented ice (i.e. above the limit)... 
Does it mean that then there is no additional freezing taking place?

**Reply by fridlind** [reply, marked as answer]
@jchylik Yes, that is correct. The diagnostic ice formation is intended to prevent ice number concentration from falling below the specified value when temperatures are too warm for homogeneous droplet freezing. If clouds get deep and cold enough to trigger homogeneous drop freezing in your model, that would be an additional source of ice.
